.gitignore

```diff
@@ -37,7 +37,9 @@ testem.log
 .local.env
 .local.env.prod
 .firebase
+.firebaserc
```

Comment:
For open-source project we might want to remove `.firebaserc` file:

https://firebase.google.com/docs/cli/#source_control_aliases